client/validators/profile.ts

```diff
@@ -11,6 +11,8 @@ export const formProfile = ({field, value, profile, errors, messages}) => {
 
     const fieldValue = (typeof value === 'string') ? value.trim() : value;
 
+    const field_value = (diff?.translations || []).filter((e) => e.field === field);
```

Comment:
By convention we use camel case for variable names in Typescript/Javascript.
Also, as this value is not used anywhere except inside the `if ` statement for multilinugal, might be better to place it there (otherwise it's filtering translations when it may not need to be used).